qutebrowser/mainwindow/prompt.py

```diff
@@ -445,6 +447,21 @@ def prompt_item_focus(self, which):
         except UnsupportedOperationError:
             pass
 
+    @cmdutils.register(instance='prompt-container', scope='window',
+                       modes=[usertypes.KeyMode.prompt])
+    @cmdutils.argument('which', choices=['next', 'prev'])
+    def prompt_history(self, which):
+        """Shift the focus of the prompt file completion menu to another item.
+
+        Args:
+            which: 'next', 'prev'
+        """
+        assert self._prompt is not None
+        if which == 'prev':
+            self._prompt.history_prev()
+        elif which == 'next':
+            self._prompt.history_next()
```

Comment:
please add a

```suggestion
            self._prompt.history_next()
        else:
            raise utils.Unreachable(which)
```

here.